src/drug_detection_crawler/storage/save_tools.py

```
@@ -325,6 +329,13 @@ def save_json_to_csv(file_path, saved_file_name):
 
     for item in json_data:
         normalized = normalize_json_item(item)
+
+        if not contains_korean(normalized.get("text", "")):
+            skippted_count += 1
```

Ruff (0.15.9)

[error] Undefined name `skippted_count`

(F821)